docs/examples/eandt.py

```diff
@@ -0,0 +1,70 @@
+from gladier import GladierBaseClient, generate_flow_definition, GladierBaseTool
+from pprint import pprint
+
+
+def makeFiles(**data):
+    import os
+    input_path = data['make_input']
+
+    if '~' in input_path:
+        input_path = os.path.expanduser(input_path)
```

Comment:
Will this work if the new path is a directory that doesn't exist? It also might be safer to run inside a newly created directory so the Tar step doesn't pick up unexpected files.

Reply:
if the expanduser function fails, the path returns unchanged, so i think that should be fine.